### PR TITLE
SC-5201 - fix-navbar-items

### PR DESCRIPTION
**Pull request by ABialoskorski**
# Description

## Links to Tickets or other pull requests

- https://github.com/schul-cloud/schulcloud-server/pull/????
- https://ticketsystem.schul-cloud.org/browse/SC-5201

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by dominikbruechner** [comment]
@schul-cloud-bot update screenshots

File: src/components/legacy/NavigationBar.vue [hunk 1]
```diff
@@ -34,8 +39,9 @@
 </template>
 <script>
 import BaseButton from "../base/BaseButton";
```
Comment (luizzaly) [inline src/components/legacy/NavigationBar.vue:41]: I think you don't need to import BaseButton or BaseLink as it is located in /base

Comment (ABialoskorski) [reply src/components/legacy/NavigationBar.vue:41]: I need them, Navbar is located in legacy, and sont see BaseButton/Link without import

Comment (luizzaly) [reply src/components/legacy/NavigationBar.vue:41]: okay makes sense :)

**Pull request merged**